.github/workflows/build.yml

```diff
@@ -144,3 +144,7 @@ jobs:
           python scripts/yaml_check.py docs/100.yaml docs/overview.yaml docs/undergrad.yaml
           bash scripts/mk_all.sh
           lean --run scripts/yaml_check.lean
+
+      - name: check for copyright headers and module docstrings
+        run: |
```

Comment:
Can this be run before any of the really slow build steps? I don't want to wait an hour for a build, only for it to then say my line is too long and have to start all over again.

Reply:
My reasoning was the opposite; since fixing the issues caught by this script shouldn't typically create new build / linter issues, it's better to report those "more serious" problems first so that those can be addressed / discussed. For example, fixing a build / linter issue (e.g. from a github suggestion) might temporarily make a line too long, and I think it'd be better to first check that the suggestion "works" rather than force people to reflow everything up front.

Feel free to bring this up on Zulip, I'd be curious to hear others' thoughts on this as well.

Note that you should be able to run this script manually by executing `./scripts/lint-copy-mod-doc.sh`.